### DISCUSSION TITLE
Nginx конфиг.

### DISCUSSION THREAD

**Discussion by evgeniy-norenko** [maxsite/cms#452] [Q&A]
Я использую вместо Апача ,Nginx можно для него конфиг ? При конвертации  правил перезаписи Apache в Nginx  вызывает ошибку .

**Comment by KsenZ** [comment]
У меня такой конфиг. Все работает.

    location / {
        try_files $uri $uri/ /index.php?$query_string;
    }

    location ~ \.php$ {
        fastcgi_split_path_info ^(.+\.php)(/.+)$;
        try_files $uri =404;
        fastcgi_pass unix:/var/run/php-fpm/php-fpm.sock;
        fastcgi_index index.php;
        fastcgi_param SCRIPT_FILENAME $request_filename;
        fastcgi_intercept_errors on;
        include fastcgi_params;
    }
